spec/swagger_helper.rb

```diff
@@ -507,6 +507,1027 @@
                             type: "string",
                             format: "date",
                           },
+                          housing_cost_type: {
+                            type: :string,
+                            example: "rent",
+                            description: "required when category is 'rent_or_mortgage'",
+                          },
+                        },
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+          Dependant: {
+            type: :object,
+            required: %i[date_of_birth in_full_time_education relationship],
+            properties: {
+              date_of_birth: {
+                type: :string,
+                format: :date,
+                example: "1992-07-22",
+              },
+              in_full_time_education: {
+                type: :boolean,
```

Comment:
slightly confused by all these changes. I thought most of these already existed?

Reply:
@starswan Not sure why github is showing these changes as a v6 change